mopro-core/src/middleware/circom/mod.rs

```diff
@@ -290,14 +293,14 @@ mod tests {
 
         // Prepare inputs
         let mut inputs = HashMap::new();
-        let a = 3;
-        let b = 5;
-        let c = a * b;
-        inputs.insert("a".to_string(), vec![BigInt::from(a)]);
-        inputs.insert("b".to_string(), vec![BigInt::from(b)]);
+        let field = "21888242871839275222246405745257275088548364400416034343698204186575808495616";
+        let a = field;
+        let b = "1";
+        let c = field;
+        inputs.insert("a".to_string(), vec![BigInt::from_str(a).unwrap()]);
+        inputs.insert("b".to_string(), vec![BigInt::from_str(b).unwrap()]);
         // output = [public output c, public input a]
-        let expected_output = vec![Fr::from(c), Fr::from(a)];
-        let serialized_outputs = SerializableInputs(expected_output);
+        let serialized_outputs = deserialize_inputs(vec![c.to_string(), a.to_string()]);
```

Comment:
Why this change? Shouldn't it still be something Serializable?

Reply:
Just to simplify the serialization
or we will need
```rust
Fr::from(BigUint::from_str(a).unwrap()))
```
to construct the element from string